st2common/st2common/util/sandboxing.py

```diff
@@ -137,6 +137,26 @@ def get_sandbox_python_path_for_python_action(
 
     pack_base_path = get_pack_base_path(pack_name=pack)
     virtualenv_path = get_sandbox_virtualenv_path(pack=pack)
+    python_binary = cfg.CONF.actionrunner.python_binary
```

Comment:
OK. I see that this is already defined. I'll just leave this note here for the next time I review this so I don't have to look it up again.

https://github.com/StackStorm/st2/blob/f5c0c0fcac41e743460de976723dc1decb144578/st2common/st2common/config.py#L454-L458